Tasks/DownloadSecureFile/task.json

```diff
@@ -31,6 +31,14 @@
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Select the secure file to download to a temporary location on the agent. The file will be cleaned up after the build or release."
+        },
+        {
+            "name": "secureFilePath",
+            "type": "secureFile",
+            "label": "Secure File Path",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Specify a custom location to download the secure file to."
```

Comment:
Can you please clarify this helpMarkDown as follows?

"Optionally specify a custom location to which the secure file will be downloaded. If empty, the agent's temp directory will be used. The temp directory path is available in the environment variable `Agent.TempDirectory`.